Plugins/DeleteUnusedBranches/DataGridViewCheckBoxHeaderCell.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Windows.Forms.VisualStyles;
+
+namespace DeleteUnusedBranches
+{
+    public class DataGridViewCheckBoxHeaderCell : DataGridViewColumnHeaderCell
+    {
+        private Point _checkBoxLocation;
+        private Size _checkBoxSize;
```

Comment:
any reason it is not a local variable?

Reply:
Yes, it is calculated inside the `Paint` method and used inside the `OnMouseClick` method.